macros/helpers/nozzle_cleaning.cfg

```diff
@@ -79,3 +79,26 @@ gcode:
     G4 P{20 * 1000}
   
     G92 E0
+
+
+[gcode_macro MOVE_TO_PURGE_BUCKET]
+description: Move over the purge bucket
+gcode:
+    {% set purge_and_brush_enabled = printer["gcode_macro _USER_VARIABLES"].purge_and_brush_enabled %}
+    {% set Z_DROP = params.Z_DROP|default(1)|int %}
+    
+    {% if purge_and_brush_enabled %}
+        SAVE_GCODE_STATE NAME=MOVE_TO_PURGE_BUCKET
+        G90
+        {% set St = printer["gcode_macro _USER_VARIABLES"].travel_speed * 60 %}
+
+        # Move to purge zone only if it's available, else just purge whe the toolhead is
+        {% set Px, Py, Pz = printer["gcode_macro _USER_VARIABLES"].purge_bucket_xyz|map('float') %}
+        {% if Z_DROP == 1 %}
+            G1 X{Px} Y{Py} Z{Pz} F{St}
+        {% else %}
+            G1 X{Px} Y{Py} F{St}
+        {% endif %}
+        RESTORE_GCODE_STATE NAME=MOVE_TO_PURGE_BUCKET
+    {% endif %}
+    
```

Comment:
Please add a newline at the end of the file :)

Reply:
I hope it done, because github desktop suppress them sometimes....